src/main/java/com/knightcharacter/app/gateway/privatedb/representation/Skill.java

```diff
@@ -0,0 +1,49 @@
+package com.knightcharacter.app.gateway.privatedb.representation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+@Table(name = "skill")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Skill {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    @ColumnDefault("random_uuid()")
+    @Column(name = "id")
+    private String id;
+
+    private String name;
+
+    private String description;
+
+    @ManyToMany
+    @JoinTable(
+        name = "bonuses_on_skills",
```

Comment:
Why do you name it like "bonuses_on_skill", why not just "bonuses"?

Reply:
Actually idk :) it seemed more informative to me to name it like "bonuses_on_skills". But I also like "bonuses_skills". Is the second option good enough?

Reply:
Maybe the best one will be "bonus_skill", because we already have tables "bonus" and "skill", not  the same words in plural

Reply:
I think that we should decide what type of naming we'll use here. I like bonus_skill. Forgot that it's a mapping table but first name wasn't good enough I think )

Reply:
like it, sure